docs/servo_power_test/src/test_runner.ts

```diff
@@ -28,26 +28,18 @@ export class TestRunner {
     const chunk = await this.dut.read();
     return chunk;
   }
-  public async copyScriptToDut(customScript: string) {
-    const script = `#!/bin/bash -e
-function workload () {
+  public async runWorkload(currentConfigNum: number, customScript: string) {
+    const script = `function workload () {
+  echo "current config: ${currentConfigNum}"
   ${customScript}
 }
 sleep 3
 echo "start"
-workload 1> ./test_out.log 2> ./test_err.log
+workload 1>> ./test_out.log 2>> ./test_err.log
```

Comment:
When will the test_{out,err}.log be deleted? Storage space on a DUT is a limited resource, so we have to care about the lifecycle of these logs. Maybe we can defer the change to the follow-up pull requests.

Reply:
I will fix this in another PR later🙇

Reply:
As discussed in our 1:1, I just deleted this redirect in this PR! I will add implementation to delete the files in another PR.